src/services/fdp-storage/fdp-storage.provider.ts

```diff
@@ -41,7 +41,9 @@ export abstract class FdpStorageProvider extends AsyncConfigService<FdpStorage>
       }
     }
 
-    return new FdpStorage(beeApiUrl, beeDebugApiUrl, options as unknown)
+    // TODO cannot cast to BatchId because it's not exported
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return new FdpStorage(beeApiUrl, process.env.POSTAGE_BATCH_ID as any, options as unknown)
```

Comment:
postage batch id should be handled by blossom only, use only zeros for batch id here